lib/themes/dosomething/paraneue_dosomething/theme-settings.php

```diff
@@ -272,46 +272,59 @@ function _paraneue_dosomething_theme_settings_footer(&$form, $form_state) {
   );
 
   $links = &$form['footer']['links'];
-  $columns = array('first', 'second', 'third');
-  foreach ($columns as $column) {
-    $prefix = 'footer_links_' . $column;
 
-    $links[$prefix] = array(
+  // Set up theme settings for each country's footer
+  $countries = dosomething_global_get_countries();
+  foreach($countries as $country) {
+    $links[$country. '_links'] = [
       '#type' => 'fieldset',
-      '#title' => t(ucwords($column) .' column'),
+      '#title' => t('@country Links', ['@country' => $country]),
       '#collapsible' => TRUE,
-      '#collapsed' => TRUE
-    );
-
-    $link_column = &$links[$prefix];
-
-    $link_column[$prefix . '_column_heading'] = array(
-      '#type' => 'textfield',
-      '#title' => t(ucwords($column) . ' Column Heading'),
-      '#default_value' => theme_get_setting('footer_links_' . $column . '_column_heading')
-    );
-
-    $link_column[$prefix . '_column_links'] = array(
-      '#type' => 'textarea',
-      '#title' => t(ucwords($column) . ' Column Links'),
-      '#default_value' => theme_get_setting('footer_links_' . $column . '_column_links')
-    );
-
-    $link_column[$prefix. '_advanced'] = array(
-      '#type' => 'fieldset',
-      '#title' => t('Advanced options'),
-      '#collapsible' => TRUE,
-      '#collapsed' => TRUE
-    );
-
-    $link_column[$prefix . '_advanced'][$prefix . '_column_class'] = array(
-      '#type' => 'textfield',
-      '#title' => t(ucwords($column) . ' Column Class'),
-      '#default_value' => theme_get_setting('footer_links_' . $column . '_column_class')
-    );
+      '#collapsed' => TRUE,
+    ];
+
+    // Set up theme settings for each of the three columns
+    $columns = ['first', 'second', 'third'];
+    foreach ($columns as $column) {
+      $prefix = 'footer_links_' . $country . '_' . $column;
+
+      $links[$country. '_links'][$prefix] = [
+        '#type' => 'fieldset',
+        '#title' => t(ucwords($column) .' column'),
+        '#collapsible' => TRUE,
+        '#collapsed' => TRUE,
+      ];
+
+      $link_column = &$links[$country. '_links'][$prefix];
+
+      // Get new country code setting if it exists, if not fallback to old setting.
+      // @TODO: Remove this after we've updated theme settings everywhere.
+      $heading_default = theme_get_setting('footer_links_' . $country . '_' . $column . '_column_heading');
+      if(!$heading_default === NULL) {
+        $heading_default = theme_get_setting('footer_links_' . $column . '_column_heading');
```

Comment:
since these old theme variables are going to be defunct, it might be cleaner to have an update hook move the settings to use the `us` prefix country code.